internal/thelma/state/providers/sherlock/state_exporter_writer.go

```diff
@@ -5,19 +5,19 @@ import (
 	"github.com/broadinstitute/thelma/internal/thelma/state/api/terra/filter"
 )
 
-type stateWriter struct {
+type StateExporterWriter struct {
```

Comment:
The structs were getting confusing to me because at first glance they seem self-referential--I renamed this one to be clear about how it gets used.

Reply:
good call, I named it when the functionality it handled was more limited.